src/features/collapsed_queue/feature.json

```
@@ -1,6 +1,6 @@
 {
   "title": "Collapsed Queue",
-  "description": "Shorten posts in your queue",
+  "description": "Shorten posts in your queue (or drafts)",
```

We're already using parentheses for (disabled) and (deprecated), so would prefer not to use them in feature descriptions as well

```suggestion
  "description": "Shorten posts in your queue or drafts",
```